tests/vm_test.go

```diff
@@ -1016,12 +1016,12 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			It("Should force stop a VMI", func() {
 
 				By("getting a VM with high TerminationGracePeriod")
-				newVMI := tests.NewRandomVMI()
-				gracePeriod := int64(1600)
-				newVMI.Spec.TerminationGracePeriodSeconds = &gracePeriod
-
+				newVMI := libvmi.New(
+					libvmi.WithResourceMemory("32Mi"),
+					libvmi.WithTerminationGracePeriod(int64(1600)),
```

Comment:
I do not think  you need to explicitly convert to `int64`.

Reply:
Thanks!
I removed the explicit conversion.